### PR TITLE
[orchagent]: srv6orch support for uSID

### PR DESCRIPTION
**Pull request by svshah-intel**
uSID (microsid) support in swss/orchagent
-	uSID “action” mapping to end-point behavior and end-point behavior-flavor
-	uSID actions: uN, uA, uDX and uDT

Current srv6orch already has appropriate infrastructure in place. Only missing piece, to support uSID, is actions mapping to end-point behavior and behavior-flavor. This commit is to add that. Related SAI support also already is in place.

swss/tests
-	Add a test-case for uSID "action" in test_srv6.py

Signed-off-by: shitanshu.shah@intel.com

### CODE REVIEW

File: orchagent/srv6orch.cpp [hunk 1]
```diff
@@ -423,9 +423,39 @@ bool Srv6Orch::sidEntryEndpointBehavior(string action, sai_my_sid_entry_endpoint
     {
         end_behavior = SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_B6_INSERT_RED;
     }
+    else if (action == "udx6")
```
Comment (prsunny) [inline orchagent/srv6orch.cpp:426]: This would be better aligned in a 'switch' case construct.

Comment (svshah-intel) [reply orchagent/srv6orch.cpp]: please review if new restructured code is better

**Review by prsunny — APPROVED** [review]
lgtm

**Comment by reshmaintel** [comment]
@prsunny Hi Prince, thank you for the quick review.
Including @shishao7sxm 
Thanks

**Pull request merged**